app/components/provider_interface/application_header_components/deferred_offer_component.html.erb

```diff
@@ -7,8 +7,7 @@
     You need to confirm your deferred offer.
   </p>
 
-<!--  TODO: Update the link path to provider_interface_deferred_offer_check_path-->
-  <%= govuk_button_link_to 'Confirm deferred offer', provider_interface_reconfirm_deferred_offer_path(application_choice) %>
+  <%= govuk_button_link_to 'Confirm deferred offer', provider_interface_deferred_offer_root_path(application_choice) %>
```

Comment:
After this PR then, then new process is implemented. Is there a trello card (or maybe something on the checklist?) for deleting any redundant code associated with the old process?